### PR TITLE
ES-831427_Subpage_Added_Azure_Function_for_Convert_WordtoImage

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Sub-page add Azure Functions both v1 and v4 for convert Word to Image.

Regards
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+//Saves the Word document in memory stream.
+MemoryStream inputStream = new MemoryStream();
+fs.CopyTo(inputStream);
+inputStream.Position = 0;
+
+try
+{
+    Console.WriteLine("Please enter your Azure Functions URL :");
+    string functionURL = Console.ReadLine();
+
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md:120]: Remove unwanted empty lines

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
+---
+title: Convert Word document to Image in Azure Functions v1 | Syncfusion
+description: Convert Word document to Image in Azure Functions v1 using .NET Word (DocIO) library  without Microsoft Word or interop dependencies.
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md:3]: remove empty space

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word to Image in Azure Functions v1
+
+Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to Image in Azure Functions v1**
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md:11]: Add full stop

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v4.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+//Saves the Word document in memory stream.
+MemoryStream inputStream = new MemoryStream();
+fs.CopyTo(inputStream);
+inputStream.Position = 0;
+
+try
+{
+    Console.WriteLine("Please enter your Azure Functions URL :");
+    string functionURL = Console.ReadLine();
+
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v4.md:126]: Remove unwanted empty line

Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v4.md:125]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md (hunk 3 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-Functions-v1.md:118]: updated

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Pull request merged**